ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/main.yml

```diff
@@ -17,6 +17,7 @@
 - name: Create Nagios user
   action: user name=nagios state=present shell=/bin/bash
   ignore_errors: yes
+  become: yes
```

Comment:
this shouldn't be needed. My PR forces the mac commands to run with become: true https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1665/files#diff-a3168cdf303b241f314135c7b81164266a94cf47e4252d4caeb6b9d9e7e08119R149-R150

Reply:
Removed!